packages/dx-react-grid-bootstrap3/src/templates/pager.jsx

```diff
@@ -57,7 +57,7 @@ export const Pager = ({
       </BootstrapPager>
       <span className="pull-right visible-xs" style={{ marginRight: '20px' }}>
         <span style={{ display: 'inline-block', verticalAlign: 'middle', lineHeight: '32px' }}>
-          {`${firstRow}${firstRow < lastRow ? `-${lastRow}` : ''} of ${totalCount}`}
+          {getMessage('info', { firstRow, lastRow, totalCount })}
```

Comment:
It is not clear that firstRow is an row index.

Reply:
Or {from} {to} {count}

Reply:
👍